### PR TITLE
footer: add social links

### PR DESCRIPTION
**Pull request by nahuhh**
https://32.md.monerodevs.org

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -119,6 +119,19 @@ theme:
 
 extra_css: ['overrides.css']
 
+extra:
+  social:
+    - icon: material/web
+      link: https://haveno.exchange
```
Comment (plowsof) [inline mkdocs.yml:125]: why

File: mkdocs.yml [hunk 2]
```diff
@@ -119,6 +119,19 @@ theme:
...
+    - icon: simple/torbrowser # TODO: use actual onion
+      link: http://o64zd74vzijrjchc5ighj2st57mjsvvjty3nhw5pgwj6xolxarlgn3qd.onion
```
Comment (plowsof) [inline mkdocs.yml:127]: #33 could make this functionally redundant

Comment (nahuhh) [reply mkdocs.yml:127]: Thinking of having i2p and onion next to one another

I2p doesnt have the browser feature, and i dont think 33 shows on mobile

File: mkdocs.yml [hunk 3]
```diff
@@ -129,13 +142,17 @@ markdown_extensions:
   - admonition
   - abbr
   - def_list
+  - attr_list
```
Comment (plowsof) [inline mkdocs.yml:145]:
```suggestion
```

Comment (plowsof) [inline mkdocs.yml:145]:
```suggestion
```

File: mkdocs.yml [hunk 4]
```diff
@@ -129,13 +142,17 @@ markdown_extensions:
...
   - meta
   - toc:
       permalink: true
-  - pymdownx.details
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
   - pymdownx.highlight:
       use_pygments: true         # build time highlighting, not client side
+  - pymdownx.details
```
[the remark this answers is not in the document]
Comment (nahuhh) [reply mkdocs.yml:155]: Docs say you need line 150-152 for the emojis and icons to work

[the remark this answers is not in the document]
Comment (nahuhh) [reply mkdocs.yml:155]: https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration

[the remark this answers is not in the document]
Comment (plowsof) [reply mkdocs.yml:155]: i confirmed on https://test.monerodevs.org/ which runs: https://github.com/plowsof/monero-docs/blob/6f34541d376e6ddc2ccaf9bd035819c1862d17df/mkdocs.yml#L120 without attr_list + others

[the remark this answers is not in the document]
Comment (nahuhh) [reply mkdocs.yml]: I just did the same
thanks

**Review by plowsof — APPROVED** [review]

File: mkdocs.yml [hunk 5]
```diff
@@ -137,6 +160,9 @@ markdown_extensions:
   - pymdownx.emoji:
       emoji_index: !!python/name:material.extensions.emoji.twemoji
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
+      options:
+        custom_icons:
+          - overrides
```
Comment (plowsof) [inline mkdocs.yml:165]:
```suggestion
```

File: mkdocs.yml [hunk 6]
```diff
@@ -117,7 +117,30 @@ theme:
   static_templates:
     - 404.html
 
-extra_css: ['overrides.css']
+extra_css:
+  - overrides.css
+  - overrides/buttons.css
```
Comment (plowsof) [inline mkdocs.yml:122]:
```suggestion
  - overrides/buttons.css
```

**Comment by plowsof** [comment]
overrides/.icons/i2p.svg can be deleted :+1:

**Review by plowsof — APPROVED** [review]

**Pull request merged**